nilearn/plotting/img_plotting.py

```diff
@@ -927,6 +693,7 @@ def plot_roi(roi_img, bg_img=MNI152TEMPLATE, cut_coords=None,
     return display
 
 
+@fill_doc
 def plot_prob_atlas(maps_img, bg_img=MNI152TEMPLATE, view_type='auto',
```

Comment:
Something in this docstring is causing the following error in the build_docs CI:

>sphinx.errors.SphinxWarning: /home/circleci/project/nilearn/plotting/img_plotting.py:docstring of nilearn.plotting.img_plotting.plot_prob_atlas:52:Inline interpreted text or phrase reference start-string without end-string.

Reply:
Thanks @tsalo ! This should be fixed now.